l10n_it_fatturapa_out_oss/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,7 @@
+**Italiano**
+
+Modulo tecnico di integrazione tra l10n_eu_oss e l10n_it_fatturapa_out
+
+**English**
+
+Technical module for integrating l10n_eu_oss and l10n_it_fatturapa_out
```

Comment:
Technical module for integrating l10n_eu_oss_oca and l10n_it_fatturapa_out

Reply:
grazie fatto